spec/lucky/params_spec.cr

```diff
@@ -79,6 +94,14 @@ describe Lucky::Params do
       params.get(:id).should eq "1"
     end
 
+    it "parses multipart params" do
```

Comment:
Maybe "it parses params in multipart requests"?

Reply:
👍